ui/js/values.js

```diff
@@ -103,6 +103,11 @@ treeherder.value("phAlertSummaryStatusMap", {
     BACKEDOUT: { id: 8, text: "backedout" }
 });
 
+treeherder.value("phAlertSummaryIssueTrackersMap", {
+    BUGZILLA: { id: 0, text: "Bugzilla", issueTrackerUrl: "https://bugzilla.mozilla.org/show_bug.cgi?id=" },
+    GITHUB: { id: 1, text: "Github", issueTrackerUrl: "https://github.com/servo/servo/pull/" }
```

Comment:
Perhaps this should be called GITHUB_SERVO, because what happens in the future if we need to add support for another Github repo?

Reply:
The repository definitions contain the repo type and base URL (eg https://github.com/mozilla/treeherder/blob/58e6690e645de083f0aa0430b31f02510b57cec2/treeherder/model/fixtures/repository.json#L1104) - so it seems like these could be auto-generated rather than hardcoded?

eg: The alert just stores the ticket number, and for display, does a lookup to see what that number maps to, with no need for the alert `issue_tracker` field

Reply:
@edmorley I don't think it's possible without that extra issue_tracker field. The perf alert summaries are linked to autoland/inbound/central/beta Mercurial repos; they're unaware of original github repos, because of the automatic servo merges.

Reply:
Ah I thought these were for alerts generated directly on the servo repo, but I can see why you need cross-repo handling here.